### PR TITLE
Add open graph and twitter share images.

### PR DESCRIPTION
**Pull request by lennerd**

### CODE REVIEW

File: src/app/PlaceDetailsData.js [hunk 1]
```diff
@@ -130,7 +130,9 @@ const PlaceDetailsData: DataTableEntry<PlaceDetailsProps> = {
 
       const equipmentPromise = equipmentInfoId ? fetchEquipment(equipmentInfoId, useCache) : null;
       const equipmentInfo = equipmentPromise
-        ? isServer ? await equipmentPromise : equipmentPromise
```
Comment (opyh) [inline src/app/PlaceDetailsData.js]: I think nested ternary operators are a bit difficult to understand – how about this? :)

```javascript
(isServer ? await equipmentPromise : equipmentPromise) || null
```

File: src/app/PlaceDetailsData.js [hunk 2]
```diff
@@ -223,48 +226,65 @@ const PlaceDetailsData: DataTableEntry<PlaceDetailsProps> = {
...
               property="place:location:latitude"
               key="place:location:latitude"
-            />
-          ),
-          coordinates != null && (
+            />,
             <meta
               content={coordinates[0]}
               property="place:location:longitude"
               key="place:location:longitud"
```
Comment (opyh) [inline src/app/PlaceDetailsData.js]: There is an `e` missing here

File: src/app/PlaceDetailsData.js [hunk 3]
```diff
@@ -223,48 +226,65 @@ const PlaceDetailsData: DataTableEntry<PlaceDetailsProps> = {
         // translator: Title for sharing a place detail page
         const thisPlaceIsOn = t`This place is on ${textContent.product.name}: ${placeTitle}`;
 
-        extras = [
-          coordinates != null && (
+        if (coordinates) {
```
Comment (opyh) [inline src/app/PlaceDetailsData.js]:
```suggestion
        if (coordinates) {
          extras.push(...['longitude', 'latitude'].map((coord, i) => <meta
              content={coordinates[i]}
              property={`place:location:${coord}`}
              key={`place:location:${coord}`}
            />))
        }
```

**Comment by lennerd** [comment]
@opyh Thanks for your feedback. Changes added.

**Pull request merged**